clientv3/watch.go

```diff
@@ -544,10 +549,14 @@ func (w *watchGrpcStream) run() {
 				w.resuming = append(w.resuming, ws)
 				if len(w.resuming) == 1 {
 					// head of resume queue, can register a new watcher
-					wc.Send(ws.initReq.toPB())
+					if err := wc.Send(ws.initReq.toPB()); err != nil {
+						plog.Warningf("when sending request, encountered: %v", err)
+					}
 				}
 			case *progressRequest:
-				wc.Send(wreq.toPB())
+				if err := wc.Send(wreq.toPB()); err != nil {
+					plog.Warningf("when sending request, encountered: %v", err)
```

Comment:
I prefer something like `"error when sending progress request: %v", err`. The current error message is very generic. When we see this warning, it is hard to know which line printed it.